src/components/test/Feed.test.tsx

```diff
@@ -91,5 +110,43 @@ describe("Feed", () => {
       button.simulate("click");
       expect(component.find(Post).length).toEqual(3);
     });
+
+    describe("Another User's Posts", () => {
+      it("Displays nav name in feed nav", async () => {
+        const component = mount(componentWithStore(Feed, store));
+        const postProfileBlock = component.findWhere((node) =>
+          node.hasClass("Post__metaBlock")
+        );
+        postProfileBlock.first().simulate("click");
+        try {
+          await waitFor(() => {
+            expect(
+              component.find(".Feed__navigationItem--active").text()
+            ).toEqual(displayProfile.name + "'s Posts");
+          });
+        } catch (e) {
+          console.log("error", component.debug());
```

Comment:
This try/catch is for debugging purposes. There's no need for it to stay in the test.